apps/examples/mediaplayer/mediaplayer_main.cpp

```diff
@@ -321,22 +316,25 @@ class MediaPlayerController
 	void start()
 	{
 		while (true) {
-			auto test = selectSource();
-			if (test < 0) {
+			vector<string> sourceList = {"Exit APP", "Test PCM", "Test BUFFER", "Test HTTP"};
+			listDirEntries("/rom", sourceList);
+			auto test = view.selectSource(sourceList);
+			if (test == 0) {
 				break;
+			} else if (test >= 4) {
```

Comment:
How about replacing "4" as the size of "sourceList"?

Reply:
sourceList was appended in listDirEntries. So the size of sourceList is not 4

Reply:
Hmm.. 4 can be stored to an integer variable at line 320 similar to line 404 (int offset = 4;) of the old source code.

Reply:
@chanijjani As per my understanding, 4 means that basic sourcelist, sourcelist append contents while it read contents in directory..
This will be enough answer for you.